charts/priobike-cache/values.yaml

```diff
@@ -0,0 +1,10 @@
+replicas: 1
+successStoreTime: 1d
+notFoundStoreTime: 1d
+imageName: priobike-cache
```

Comment:
```suggestion
image:
     name: priobike-cache
     version: v1.4
```

Reply:
If the version should be configurable, we could do it like this

Reply:
Should not be configurable since the version may also be directly coupled to the chart configuration (eg env variables or other stuff). The version is also set in the `Chart.yaml` file and should therefore be hardcoded.